doc/user/content/manage/user-management/rbac-alterroles.md

```diff
@@ -0,0 +1,153 @@
+---
+title: "Update and drop roles"
+description: ""
+menu:
+  main:
+    parent: user-management
+    weight: 16
+---
+
+In the previous guide, you created a new role and assigned privileges for
+specific objects. This guide walks you through updating your previously created
+roles and removing users or roles when they are no longer needed.
+
+## Before you begin
+
+In this guide, you'll use `psql` to manage users in Materialize, so be sure you
+have `psql` installed locally.
+
+If you have not completed the [New User guide](/manage/user-management/rbac-newrole.md), you'll need to
+create an example environment with a new user, new role, cluster, database,
+schema, and table.
+
+## Create a second role
```

Comment:
As mentioned in the other guide, since these sections are a sequence of steps, I think we should be explicit about that and use `Step 1. Create a second role`, `Step 2. Add inherited privileges to a role`, etc.